modules/refreshList.js

```diff
@@ -0,0 +1,7 @@
+import displayLists from './displayLists.js';
+
+export default function refreshList(toDoList) {
```

Comment:
- [ ]  Kindly check my comment on the use of es6 on the addToList file which is very detailed 👍